src/GitExtensions.GerritPlugin/GitExtensions.GerritPlugin.nuspec

```diff
@@ -11,11 +11,11 @@
     <icon>Resources\IconGerrit.png</icon>
     <dependencies>
       <group>
-        <dependency id="GitExtensions.Extensibility" version="[0.3.0, 0.4.0)" />
+        <dependency id="GitExtensions.Extensibility" version="[0.3.1, 0.4.0)" />
       </group>
       <!-- To fix Warning NU5128 Add a dependency group for net6.0-windows7.0 to the nuspec -->
       <group targetFramework="net6.0-windows7.0">
-        <dependency id="GitExtensions.Extensibility" version="[0.3.0, 0.4.0)" />
+        <dependency id="GitExtensions.Extensibility" version="[0.3.1, 0.4.0)" />
```

Comment:
Should it be still 0.3.0 for the existing GitExtensions v4.2? Looks like the newly released version of the plugin (2.0.7) can't be discovered by the GE Plugin Manager for some reason.

Reply:
Yes. I've only changed them as I was struggling to understand how to get the 4.2 deps.
It took me a while to just read the msbuild targets in order to clarify this.